net/flutterfire.dart

```diff
@@ -44,9 +44,10 @@ Future<bool> addCoin(String id, String amount) async {
       if (!snapshot.exists) {
         documentReference.set({'Amount': value});
         return true;
+      } else {
+        double newAmount = snapshot.data()['Amount'] + value;
```

Comment:
could somebody pls make the null safety version of this entire method?? I fixed some issues but on this line I can't continue....

Reply:
double newAmount = (snapshot.data() as Map)['Amount'] + value;